detections/endpoint/linux_auditd_add_user_account.yml

```diff
@@ -0,0 +1,63 @@
+name: Linux Auditd Add User Account
+id: aae66dc0-74b4-4807-b480-b35f8027abb4
+version: 1
+date: '2024-08-27'
+author: Teoderick Contreras, Splunk
+status: production
+type: Anomaly
+description: The following analytic detects the creation of new user accounts on Linux
+  systems using commands like "useradd" or "adduser." It leverages data from Endpoint
+  Detection and Response (EDR) agents, focusing on process names and command-line
+  executions. This activity is significant as adversaries often create new user accounts
+  to establish persistence on compromised hosts. If confirmed malicious, this could
+  allow attackers to maintain access, escalate privileges, and further compromise
+  the system, posing a severe security risk.
+data_source:
+- Linux Auditd Proctitle
+search: '`linux_auditd` `linux_auditd_normalized_proctitle_process`| rename host as dest | where LIKE (process_exec,
+  "%useradd%") OR LIKE (process_exec, "%adduser%") | stats count min(_time) as firstTime
+  max(_time) as lastTime by process_exec proctitle dest  | `security_content_ctime(firstTime)`|
+  `security_content_ctime(lastTime)`| `linux_auditd_add_user_account_filter`'
+how_to_implement: The detection is based on data that originates from Endpoint Detection
+  and Response (EDR) agents. These agents are designed to provide security-related
+  telemetry from the endpoints where the agent is installed. To implement this search,
+  you must ingest logs that contain the process complete command-line executions.
+  These logs must be processed using the appropriate Splunk Technology Add-ons that
+  are specific to the EDR product. Use the Splunk Common Information Model (CIM) to
+  normalize the field names and speed up the data modeling process.
+known_false_positives: Administrator or network operator can execute this command.
+  Please update the filter macros to remove false positives.
+references:
+- https://linuxize.com/post/how-to-create-users-in-linux-using-the-useradd-command/
+tags:
+  analytic_story:
+  - Linux Privilege Escalation
+  - Linux Persistence Techniques
+  - Compromised Linux Host
+  asset_type: Endpoint
+  confidence: 50
+  impact: 50
+  message: $process_exec$ event occurred on host.
```

Comment:
Can we improve this message to give more details as per the description something like :

New user added using process - $process_exec$ on host - $dest$